File: generated-sources/javascript-closure-angular/mojang-api/API/Client/OrderStatisticsRequest.js
goog.provide('API.Client.OrderStatisticsRequest');

/**
 * The body of the request to get the order statistics
 * @record
 */
API.Client.OrderStatisticsRequest = function() {}

/**
 * @type {!Array<!API.Client.OrderStatistic>}
 * @export
 */
API.Client.OrderStatisticsRequest.prototype.metricKeys;

File: generated-sources/scala-finch/mojang-api/src/main/scala/com/github/asyncmc/mojang/api/scala/finch/model/OrderStatisticsRequest.scala
package com.github.asyncmc.mojang.api.scala.finch.model

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._
import com.github.asyncmc.mojang.api.scala.finch.model.OrderStatistic
import scala.collection.immutable.Seq

/**
 * The body of the request to get the order statistics
 * @param metricKeys 
 */
case class OrderStatisticsRequest(metricKeys: Seq[OrderStatistic]
                )

object OrderStatisticsRequest {
    /**
     * Creates the codec for converting OrderStatisticsRequest from and to JSON.
     */
    implicit val decoder: Decoder[OrderStatisticsRequest] = deriveDecoder
    implicit val encoder: ObjectEncoder[OrderStatisticsRequest] = deriveEncoder
}
